### PR TITLE
use streams

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

**Comment by gamerson** [comment]
hey @jtydhr88  can you re-test and ensure I didn't break anything with my review. :)

File: src/main/java/com/liferay/ide/idea/language/osgi/ComponentPropertiesCompletionContributor.java [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+	}
+
+	//see https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/portlet-descriptor-to-osgi-service-property-map
+	private static Map<String, String[][]> _componentProperties = new HashMap<>();
+
+	static {
+		_componentProperties.put(
+			"javax.portlet.Portlet",
+			new String[][] {
+				{"javax.portlet.description", "String"}, {"javax.portlet.name", "String"},
```
Comment (gamerson) [inline src/main/java/com/liferay/ide/idea/language/osgi/ComponentPropertiesCompletionContributor.java]: @jtydhr88  can you sort all of these when you send the PR back?  thanks!

File: src/main/java/com/liferay/ide/idea/language/osgi/ComponentPropertiesCompletionContributor.java [hunk 2]
```diff
@@ -0,0 +1,523 @@
...
+				{"com.liferay.portlet.footer-portal-javascript", "String"},
+				{"com.liferay.portlet.footer-portlet-javascript", "String"},
+				{"com.liferay.portlet.css-class-wrapper", "String"},
+				{"com.liferay.portlet.facebook-integration", "String"},
+				{"com.liferay.portlet.add-default-resource", "boolean"}, {"com.liferay.portlet.system", "boolean"},
+				{"com.liferay.portlet.active", "boolean"}, {"com.liferay.portlet.display-category", "String"}
+			});
+
+		//see https://dev.liferay.com/de/develop/tutorials/-/knowledge_base/7-0/mvc-action-command
+		_componentProperties.put(
```
Comment (gamerson) [inline src/main/java/com/liferay/ide/idea/language/osgi/ComponentPropertiesCompletionContributor.java]: can you sort the calls to `put` as well?  Thanks!

**Comment by jtydhr88** [comment]
sent back

**Pull request closed without merging**